app/api/db/models/base.py

```diff
@@ -33,14 +34,8 @@ def for_json(self):
 
         return json_valid_dict
 
-    def __rich_repr__(self):
-        """Rich repr for interactive console.
-
-        See https://rich.readthedocs.io/en/latest/pretty.html#rich-repr-protocol
-        """
-        return self.dict().items()
-
-    def copy(self, **kwargs):
+    def copy(self, **kwargs: dict[str, Any]) -> "Base":
```

Comment:
The method returns an instance of the class `Base`, but mypy can't handle that as the class isn't yet defined (you're actively defining it), so you put it in quotes. Similar to when two classes reference each other.

Note python 3.11 will add `Self` to the typing library so this could be `-> Self` instead.

Reply:
Oh right. Thank you!